src/kvstore/RocksEngine.cpp

```diff
@@ -363,6 +363,10 @@ ResultCode RocksEngine::setDBOption(const std::string& configKey, const std::str
 
 ResultCode RocksEngine::compact() {
     rocksdb::CompactRangeOptions options;
+    // If change_level is true and target_level is -1,
+    // compacted files will be moved to the minimum level capable of holding the data.
+    options.change_level = true;
+    options.target_level = -1;
```

Comment:
You'd better add a default parameter, by default use `CompactRangeOptions()`

Reply:
ok!